src/player_team.ts

```diff
@@ -1125,15 +1140,16 @@ class Team {
       }
 
       let val = ping.damage;
-      val = Math.fround(val) * Math.fround(partialAtk(leadId, ping, healing) * 100) / Math.fround(100);
-      val = Math.fround(val) * Math.fround(partialAtk(helpId, ping, healing) * 100) / Math.fround(100);
+      val = Math.fround(val) * Math.fround(partialAtk(leadId, ping, healingFromCombos) * 100) / Math.fround(100);
+      val = Math.fround(val) * Math.fround(partialAtk(helpId, ping, healingFromCombos) * 100) / Math.fround(100);
       mults[i].badge = atkBadgeMult;
-      mults[i].lead = partialAtk(leadId, ping, healing);
-      mults[i].help = partialAtk(helpId, ping, healing);
+      mults[i].lead = partialAtk(leadId, ping, healingFromCombos);
+      mults[i].help = partialAtk(helpId, ping, healingFromCombos);
       ping.damage = Math.round(val * atkBadgeMult);
       mults[i].final = ping.damage;
     }
 
+    let healing = -poison + healingFromCombos;
```

Comment:
Why not healingFromCombos - poison?

Reply:
I wrote it in the order that it happens in-game since that seemed more intuitive to parse, but I'm apathetic about that formulation.